pytket/binders/circuit/Circuit/main.cpp

```diff
@@ -431,7 +431,9 @@ void def_circuit(py::class_<Circuit, std::shared_ptr<Circuit>> &pyCircuit) {
           "from the circuit. This may occur when optimisations "
           "recognise that the operations on a qubit reduce to the "
           "identity, or when routing adds wires to \"fill out\" the "
-          "architecture.")
+          "architecture.\n\n:param remove_classical_wire: select if "
+          "empty classical wires should be removed, too",
+          py::arg("remove_classical_wire"))
```

Comment:
Suggest having a `keep_blank_classical_wires` argument that defaults to `false`.

Reply:
done in [3b4bb6f](https://github.com/CQCL/tket/pull/1435/commits/3b4bb6fe63ad1d197206e29f0b668c41d2d599df)